### PR TITLE
Fix false ////- and /***-natspec

### LINKED ISSUE #9139: Comments starting with multiple stars wrongly recognized as natspec docstring
## Description

The following code generate a warning in 0.6.9:
```
/***** STORAGE *****/
MyStruct storage data = MyStruct(0);
```

```
> Compilation warnings encountered:

    /home/node/project/contracts/MyContract.sol:50:18: Warning: Only state variables can have a docstring. This will be disallowed in 0.7.0.
    MyStruct storage data =
                  ^------^
````

The error message might as well be improved and enlight the natspec comments instead.

## Environment

- Compiler version: starting from 0.6.9
- Target EVM version (as per compiler settings): 0.6.9
- Framework/IDE (e.g. Truffle or Remix): truffle

### PR DESCRIPTION
**Pull request by a3d4**
Fixes #9139.

### CODE REVIEW

File: liblangutil/Scanner.cpp [hunk 1]
```diff
@@ -313,7 +313,6 @@ size_t Scanner::scanSingleLineDocComment()
 {
 	LiteralScope literal(this, LITERAL_TYPE_COMMENT);
 	size_t endPosition = m_source->position();
```
Comment (a3d4) [inline liblangutil/Scanner.cpp:315]: Previously, we called `position()` before `advance()`. If the source ended with `///` we captured `//` only.

File: test/libsolidity/syntaxTests/natspec/false_natspec.sol [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    /***/
+    S g;
+    /****/
+    S h;
+}
+// ----
+// Warning: (117-144): Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
+// Warning: (158-185): Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
+// Warning: (199-254): Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
+// Warning: (268-305): Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
```
Comment (hrkrshnn) [inline test/libsolidity/syntaxTests/natspec/false_natspec.sol]: Do you think adding a test in `test/libsolidity/SolidityNatspecJSON.cpp` makes sense?

**Review by hrkrshnn — COMMENTED** [review]
LGTM. Needs a changelog though.

File: test/libsolidity/syntaxTests/natspec/false_natspec.sol (hunk 2 above)
Comment (chriseth) [reply test/libsolidity/syntaxTests/natspec/false_natspec.sol]: I agree, this is very hard to review like that.

**Comment by chriseth** [comment]
Still needs a changelog entry.

File: Changelog.md [hunk 3]
```diff
@@ -8,8 +8,7 @@ Compiler Features:
 
 
 Bugfixes:
-
-
+ * NatSpec: Do not consider //// and /*** as natspec comments.
```
Comment (hrkrshnn) [inline Changelog.md]:
```suggestion
 * NatSpec: Do not consider ``////`` and ``/***`` as NatSpec comments.
```

**Review by chriseth — APPROVED** [review]

**Pull request merged**